### DISCUSSION TITLE
js/bundle-blog.js

### DISCUSSION THREAD

**Discussion by a0damasco** [kdevo/osprey-delight#31] [Q&A] [answered]
Hi, I love the theme!
body-bottom.html line 25: "js/bundle-blog.js" , but that file is missing... or is the code unnecessary? or is it correct as it is?
I ask out of curiosity, I'm not a programmer and I'm trying to understand how this works :)

**Comment by kdevo** [comment, marked as answer]
Hi @a0damasco,

It's correct as is.

https://github.com/kdevo/osprey-delight/blob/9b1fea2b45dc73e9ee45b477ea3011cb879de510/layouts/partials/body-bottom.html#L23-L26
This file is bundling all blog-relates JS files to a `bundle-blog.js` file.

The `$scripts | resources.Concat "js/bundle-blog.js"` means "take all $scripts and concat them to a file named `js/bundle-blog.js`.

Doc reference: https://gohugo.io/hugo-pipes/bundling/

**Reply by a0damasco** [reply]
Thanks!
I really like the theme.

**Reply by kdevo** [reply]
Glad that you like it. I've just released a new version, feel free to add your feedback to [this release discussion](https://github.com/kdevo/osprey-delight/discussions/34).
